install.sh

```diff
@@ -777,9 +779,17 @@ get_free_range() {
 set_user_subuids_subgids() {
 
   local min_range_size=65536
-  local current_user=$(get_user)
+  local current_user="${OPT_USER:-$(get_user)}"
   local free_range=$(get_free_range)
 
+  if [[ "$current_user" == "root" ]]; then
+    abort "You cannot assign subuids/subgids to root."
+  fi
```

Comment:
You sure about this (I'm not)? If so, then fine. Frankly, I don't even remember whether Buildah is gonna work via root. If it's not, then I think we should just forbid running script as root with Buildah enabled, and do it as early in the script as possible, this is way too late.